src/domain/app/footer/footerCategories.module.scss

```diff
@@ -0,0 +1,40 @@
+@import 'layout';
```

Comment:
old `topFooter.module.css` with some changes but it seems that git fails to show changed rows after file rename.